cmd/destroy/destroy.go

```diff
@@ -237,11 +235,14 @@ func (dc *destroyCommand) getDestroyer(ctx context.Context, opts *Options) (dest
 	} else {
 		manifest, err := model.GetManifestV2(opts.ManifestPath)
 		if err != nil {
-			// Log error message but application can still be deleted
+			// error will only be available when there is no valid manifest found
+			// hence, we need not to destroy because the manifest would never be deployed
+			// Return the error message
 			oktetoLog.Infof("could not find manifest file to be executed: %s", err)
 			manifest = &model.Manifest{
 				Destroy: &model.DestroyInfo{},
 			}
+			return nil, err
```

Comment:
```suggestion
			// error will only be available when there is no valid manifest found
			// hence, we need not to destroy because the manifest would never be deployed
			// Return the error message
			oktetoLog.Infof("could not find manifest file to be executed: %s", err)
			manifest = &model.Manifest{
				Destroy: &model.DestroyInfo{},
			}
			return nil, err
```
I think that for now this change is not necessary. Let's keep the changes to a minimum to avoid possible regressions.

Reply:
I understand, but how would this look that you've not deployed anything on Okteto cloud and you don't have a valid manifest as well yet the `okteto destroy` command successfully destroys without having any manifest file.

Reply:
if we can't get the destroy commands from the manifest we use the deploy-by label as well as the configmap belonging to the dev environment. With your change, we would not be removing the dev environment but showing an error. However, this change is not related to the one reported in the issue. We should stick to the definition of the [scope of a PR](https://github.com/okteto/okteto/blob/master/docs/code-review.md#pr-scope) in this case to facilitate the review and possible resolution of future bugs.

Reply:
`okteto deploy` cannot be executed due to there is no manifest detected so `okteto` doesn't know how to deploy the environment. However, `okteto destroy` although it does not detect it either, it performs a destroy action removing all resources within namespace which has the proper `deployed-by` label. After that, `destroy` removes also configmap related to that environment. For that reason, `destroy` doesn't need a manifest to use the default `destroy`. Maybe we could improve our logs to let to know the user what happened clearly. If you think that we could improve our dev experience in this case please feel free to fill a new issue :smile:

Reply:
I see, that's how it's working underneath. Thanks for explaining! I shall mark these changes now.

Reply:
Resolved!